cylc/flow/scripts/view.py

```diff
@@ -18,11 +18,16 @@
 
 """cylc view [OPTIONS] ARGS
 
-Print a processed workflow configuration.
+Print workflow configurations as processed before full parsing by Cylc. This
```

Comment:
IMO, each command/config/whatever should have a concise simple description.

The first line of the CLI help is also used by `cylc help all` so needs to be concise.

```suggestion
Print a processed workflow configuration.

Print workflow configurations as processed before full parsing by Cylc. This
```

Reply:
> IMO, each command/config/whatever should have a concise simple description.

I agree. 

Problem was the original didn't say what "processed" means.  I suspect most users would equate that with "parsed".

But moving the fuller explanation down a bit is all good. (And I'd forgotten we were using the first line like that).